### PR TITLE
Markers: Save to CSV

### PR DESCRIPTION
**Pull request by pllim**
### Description

This pull request is to saving to CSV.

### Change log entry

- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [x] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [x] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone. Bugfix milestone also needs an accompanying backport label.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)? [🐱](https://jira.stsci.edu/browse/JDAT-4626)

### CODE REVIEW

File: jdaviz/configs/imviz/plugins/coords_info/coords_info.py [hunk 1]
```diff
@@ -393,6 +393,8 @@ def _image_viewer_update(self, viewer, x, y):
             self.row3_unreliable = unreliable_world
             # TODO: use sky directly, but need to figure out how to have a compatible "blank" entry
             self._dict['world'] = (sky.ra.value, sky.dec.value)
```
Comment (kecnry) [inline jdaviz/configs/imviz/plugins/coords_info/coords_info.py:395]: does it still even make sense to store world here (and same with pixel below)? Are they ever used anywhere now?

File: jdaviz/core/template_mixin.py [hunk 2]
```diff
@@ -4524,7 +4524,7 @@ def float_precision(column, item):
                     # stored in astropy table as a float so we can also store nans,
                     # but should display in the UI without any decimals
                     return f"{item:.0f}"
-                elif column in ('pixel', ):
+                elif column in ('pixel', 'pixel_x', 'pixel_y'):
```
Comment (pllim) [inline jdaviz/core/template_mixin.py:4527]: This mixin is quite generic so I left `'pixel'` in, just in case.

**Comment by pllim** [comment]
> Is this considered an API breaking change? 

Since we do not support reading this back in, I don't see this change being "breaking".

**Comment by pllim** [comment]
But I wouldn't backport it.

**Review by cshanahan1 — APPROVED** [review]
tested this out and it looks good to me

**Comment by pllim** [comment]
@kecnry , are you still concerned about the change log? Any suggestion?

File: CHANGES.rst [hunk 3]
```diff
@@ -86,6 +86,8 @@ Specviz2d
 Bug Fixes
 ---------
 
+- Markers table can now export to CSV but its columns had to be changed to accomodate this fix. [#3089]
```
Comment (kecnry) [inline CHANGES.rst:89]: How about something like this?

```suggestion
- Markers table can now export to CSV but its columns had to be changed to accomodate this fix: world and pixel (previously containing SkyCoordinate tuples) are now each two separate columns for world_ra/world_dec and pixel_x/pixel_y, respectively. [#3089]
```

**Comment by pllim** [comment]
@kecnry , okay, I accepted your suggestion and cleaned it up. Thanks! Okay for my to squash merge when CI passes?

**Pull request merged**